libs/sdk-core/src/lib/db/kb/kb.ts

```diff
@@ -31,14 +31,27 @@ import { ResourceProperties } from '../db.models'; // eslint-disable-next-line @
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface KnowledgeBox extends IKnowledgeBox {}
+
+/** Provide access to all the Knowledge Box contents and services in read mode.
+ *
+ * For any Knowledge Box operation that requires write access, you need to use `WritableKnowledgeBox` instead.
```

Comment:
is it possible to make this `WritableKnowledgeBox` navigable so interested users can directly click on it to access to corresponding documentation?